app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
please use i18n for text "Ruby on Rails Tutorial Sample App"
https://guides.rubyonrails.org/i18n.html

Reply:
Sorry i have read I am not sure how do i use it

Reply:
you should do follow this
 https://lokalise.com/blog/rails-i18n/